patches/components-sync_device_info-device_info_sync_bridge.cc.patch

```diff
@@ -10,7 +10,15 @@ index 69dbe2887fae63f3635423ea9a22c3f6ed5f7248..a9d224f072be5a5da78881300031e4ea
    return specifics;
  }
  
-@@ -439,7 +440,7 @@ absl::optional<ModelError> DeviceInfoSyncBridge::ApplySyncChanges(
+@@ -430,6 +431,7 @@ absl::optional<ModelError> DeviceInfoSyncBridge::ApplySyncChanges(
+     const std::string guid = change->storage_key();
+ 
+     // Reupload local device if it was deleted from the server.
++    if (bool false_as_brave_needs_skip_next_if = false)
```

Comment:
Seems like we can follow our patching patterns here and use `#define BRAVE_DEVICE_INFO_SYNC_BRIDGE_APPLY_SYNC_CHANGES_SKIP_NEXT_IF if (false)` in the `chromium_src` override.

Reply:
and you can also add comment in the overridden file

Reply:
thanks @mkarolin @darkdh , I missed that pattern, I'll fix

Reply:
fixed at https://github.com/brave/brave-core/compare/3bb43a4b5f4bdae7a20fbb13789578af33235bad..2ac8203b714bd992f48571119cb9b8381c59136d